articles/howtos/index.md

```diff
@@ -0,0 +1,3 @@
+# How-to guides
+
+We consider the fact that you would like to accomplish something specifically but you don't really know how to go about it. This is the section for you. Please look at the panel to the left to locate the question that you have on your mind.
```

Comment:
```suggestion
This is a collection how-to articles for guiding you through accomplishing specific tasks.

Please refer to the side panel for the available articles.
```

The second sentence could be omitted if it's possible to embed the TOC somehow.

Reply:
That would be great, yes.